common/xrdp_client_info.h

```diff
@@ -23,6 +23,8 @@
 #if !defined(XRDP_CLIENT_INFO_H)
 #define XRDP_CLIENT_INFO_H
 
+#define XRDP_MAXIMUM_MONITORS 16
```

Comment:
Whhere does this maximum number of monitors come from?

Reply:
I think this should be defined somewhere else, not in client_info at least.

Reply:
The max monitors occurs in [[MS-RDPBCGR] section 2.2.1.3.6](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/a8029f8e-01e1-4f19-af67-bcad5bdef624), although it's not given an actual name.

There's some precedent for adding it to `common/ms-rdgbcgr.h` - see `CHANNEL_NAME_LEN` in that file.

Reply:
I've added `CLIENT_MONITOR_DATA_MAXIMUM_MONITORS` to ms-rdgbcgr.h and pushed the update. Let me know what you think.